docs/pages/build/internal-distribution.mdx

```diff
@@ -3,168 +3,93 @@ title: Internal distribution
 
-<Step label="4">
+</Collapsible>
 
-## Install and run the build
+## Internal distribution with EAS Build
 
-Press the "Install" button on the build page and follow the instructions presented in the modal.
+Setting up an internal distribution build only takes a few minutes with EAS Build and provides a streamlined way to share your app with your team and testers for feedback.
 
-If you're running iOS 16 or above and haven't yet turned on Developer Mode, you will need to [enable it](/guides/ios-developer-mode) before you can run your build. This doesn't apply if you're using enterprise provisioning.
+The setup requires configuring a build profile in the **eas.json** configuration file and setting `"distribution": "internal"` to a build profile. This configuration automatically generates **.apk** files for Android and **.ipa** files for iOS so that they can be installed on devices directly and do not require a development server to run the app.
 
-</Step>
+For Android, the setup uses the Android keystore or can re-use the previously generated keystore for a [development build](/develop/development-builds/introduction/). However, iOS requires an ad hoc provisioning profile that explicitly allows devices to run that app based on that device's UDID. This can be done using `eas device:create` when creating a new build.
 
-<Step label="5">
+See the tutorial on Internal distribution with EAS Build below for more information on how to configure, create, and install a build:
 
-## Automation on CI (optional)
+<BoxLink
+  title="Create and share internal distribution build"
+  description="Complete step-by-step guide to setting up and sharing internal distribution builds with EAS Build."
+  href="/tutorial/eas/internal-distribution-builds/"
+  Icon={GraduationHat02DuotoneIcon}
+/>
 
-It's possible to run internal distribution builds non-interactively in CI using the `--non-interactive` flag; however, if you are using ad hoc provisioning on iOS you will not be able to add new devices to your provisioning profile when using this flag. After registering a device through `eas device:create`, you need to run `eas build` interactively and authenticate with Apple for EAS to add the device to your provisioning profile. [Learn more about triggering builds from CI](/build/building-on-ci).
+### Automation on CI (optional)
 
-</Step>
+It's possible to run internal distribution builds non-interactively in CI using the `--non-interactive` flag. However, if you are using ad hoc provisioning on iOS you will not be able to add new devices to your provisioning profile when using this flag. After registering a device through `eas device:create`, you need to run `eas build` interactively and authenticate with Apple for EAS to add the device to your provisioning profile. [Learn more about triggering builds from CI](/build/building-on-ci).
```

Comment:
```suggestion
It's possible to run internal distribution builds non-interactively in CI using the `--non-interactive` flag. However, if you are using ad hoc provisioning on iOS you will not be able to add new devices to your provisioning profile when using this flag. After registering a device through `eas device:create`, you need to run `eas build` interactively and authenticate with Apple in order for EAS to add the device to your provisioning profile. [Learn more about triggering builds from CI](/build/building-on-ci).
```
I can't articulate a grammar rule that specifies this, but intuitively feels needed (prevents you from accidentally reading "Apple for EAS" as a noun, perhaps?)